tools/e2etesting/pipeline.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     inputs:
       azureSubscription: '$(AzureSubscription)'
       KeyVaultName: '$(ClientCredentialsKeyVaultName)'
-      SecretsFilter: '*'
+      SecretsFilter: 'IAIClientId,IAIClientSecret'
```

Comment:
Note the change to this line is unrelated but still a good idea to only download the secrets we need from keyvault.

Reply:
I think we should investige whether IAI can use the authentication context that Azure DevOps provides somehow so that we don't need this step at all.